modal_proto/api.proto

```diff
@@ -912,6 +912,7 @@ message EnvironmentListItem {
   string name = 1;
   string webhook_suffix = 2;
   double created_at = 3;
+  bool default = 4;
```

Comment:
Glad we went this route as it also lets us support user-configurable default envs on the server side later.